server/server.py

```diff
@@ -0,0 +1,33 @@
+import socket, time
+
+HOST = socket.gethostbyname(socket.gethostname()) #127.0.1.1
+PORT = 5050
+
+server = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+server.bind((HOST, PORT))
+clients = []  
+
+def start_server():
+    print(f"[ Server Started ] Server is listening on {HOST}:{PORT}")
```

Comment:
I suggest that all messages that can be output several times, for example, error messages should be stored in a separate variable and simply output it